client/src/util/lightMode.js

```diff
@@ -0,0 +1,29 @@
+export function shouldUseLightMode() {
+  const localStoragePreference = localStorage.getItem('isLightMode')
+
+  if (localStoragePreference !== null) return localStoragePreference === 'true'
+
+  const userOSPreference =
+    window.matchMedia &&
```

Comment:
do we need error handling here? (im guessing with most browsers this shouldn't be an issue)

Reply:
oh localstorage returns null if the item was never set (i.e. first time user of our app)